src/NosCore.MasterServer/Controllers/WarehouseController.cs

```diff
@@ -55,6 +55,14 @@ public List<WarehouseLink> GetWarehouseItems(Guid? id, long? ownerId, WarehouseT
                     => s.Type == warehouseType
                     && s.CharacterId == (warehouseType == WarehouseType.FamilyWareHouse ? null : ownerId)
                     && s.FamilyId == (warehouseType == WarehouseType.FamilyWareHouse ? ownerId : null));
+                if (slot == null)
+                {
+                    //todo add
+                }
+                else
+                {
+                    //todo add
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262030)